packages/header/src/types.ts

```diff
@@ -0,0 +1,9 @@
+export type TApplication = {
+  name: string
+  type: string
+  dataSources: string[]
+  label: string
+  description: string
+  roles: string[]
+  adminRole: string
```

Comment:
These should be optional. 
Not every app will need them, but they should still be able to use the header plugin